apps/web/pages/availability/[schedule].tsx

```diff
@@ -56,6 +56,18 @@ const DateOverride = ({ workingHours }: { workingHours: WorkingHours[] }) => {
   const { remove, append, update, fields } = useFieldArray<AvailabilityFormValues, "dateOverrides">({
     name: "dateOverrides",
   });
+
```

Comment:
@sean-brydon I've implemented this to display all the dates separately in the UI, but the issue is the key of all these elements is the same so the delete/edit button does not function properly.

What could be the solution here? 🤔 

Thanks for the help again! 🙇